src/components/RowSelection/index.js

```diff
@@ -1,22 +1,59 @@
-import React, { PureComponent } from 'react';
-import PropTypes from 'prop-types';
-
+import React from 'react';
+import { Dropdown, DropdownItem, DropdownSeparator, DropdownToggle } from '@patternfly/react-core';
+import CaretDownIcon from '@patternfly/react-icons/dist/js/icons/caret-down-icon';
 import Button from '../Button';
 
-export default class RowSelection extends PureComponent {
-  static propTypes = {
-    selectedItems: PropTypes.array.isRequired,
-    compareActionName: PropTypes.string.isRequired,
-    onCompare: PropTypes.func.isRequired,
-    style: PropTypes.object,
+export default class RowSelection extends React.Component {
```

Comment:
We should consider moving the "Manage Dropdown" dropdown button to a separate component that is then reference in the `RowSelection` component. This will improve & help separate logic of both components here.

Reply:
Yes, but then it would cause too much of props drilling.